python/ray/tune/trainable.py

```diff
@@ -809,3 +811,20 @@ def _export_model(self, export_formats, export_dir):
 
     def _implements_method(self, key):
         return hasattr(self, key) and callable(getattr(self, key))
+
+
+class DistributedTrainable(Trainable):
+    """Common class for a Trainable whose training should be performed in a distributed manner."""
```

Comment:
For this to render properly on the docs, we need this to be 1 sentence that does not line wrap (called a summary sentence). 

```
"""Common Trainable class for distributed training."""

Reply:
Is there a linter for this? What's the character limit?

Reply:
80 char, you can run `ray/scripts/format.sh` i think